deployment/aks-periscope.yaml

```diff
@@ -69,7 +69,7 @@ spec:
         beta.kubernetes.io/os: linux
       containers:
       - name: aks-periscope
-        image: aksrepos.azurecr.io/staging/aks-periscope:v0.3
+        image: docker.io/tatsat/test-arc-stuff
```

Comment:
Should we leave this image as your test image or should I change this back to the periscope image as before?

Reply:
I think you are looking at the outdated commit, 🙏 see the commit after which removed this image and got the current back.